### PR TITLE
Add Bendigo Bank

### PR DESCRIPTION
**Pull request by jimmyvanau**

### CODE REVIEW

**Comment by phallobst** [comment]
> In order to access some services in Bendigo e-banking, like updating your Contact Details or sending money using Pay Anyone, a Security Token is required.

This sounds more like _authorization_, not _authentication_. Do you need an "e-token" when logging into your account?

**Comment by jimmyvanau** [comment]
This is the logon screen
image

And I've always used the token. Testing again, it seems to ignore that input for my account, as in I can leave it empty. That is interesting 🤔 Seems like it's not proper 2FA then.

**Comment by phallobst** [comment]
Could it be that entering the security token enables a "super user" mode that allows you to change account details or use "Pay Anyone", whereas leaving it blank just gives you basic functions, like checking your balance?
Or do you need to enter a token every time that you want to authorize something?

**Comment by phallobst** [comment]
@jimmyvanau Can you provide more information regarding my [comment](https://github.com/2factorauth/twofactorauth/pull/4718#issuecomment-611477101)?

**Comment by jimmyvanau** [comment]
Hi @phallobst thanks for following up - forgot about this.

Just tested it. It looks like the token is required for extra/secure actions. I can login, check balance and transfer to known users with username + pw. If I attempt to pay a new person after logging in with username + pw only, then it prompts for the token.

**Comment by Carlgo11** [comment]
So that would be closer to two step [authorization](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#a-note-on-definitions) than authentication.

File: _data/banking.yml [hunk 1]
```diff
@@ -222,6 +222,15 @@ websites:
     facebook: bbvacompass
     twitter: BBVACompass
 
+  - name: Bendigo Bank
+    url: https://www.bendigobank.com.au
+    img: bendigobank.png
+    tfa:
+      - proprietary
+    doc: https://www.bendigobank.com.au/help/security/online-security/e-tokens/
```
Comment (phallobst) [inline _data/banking.yml]: Let's list it as non-supporting then.
```suggestion
    facebook: BendigoBank 
    twitter: bendigobank
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: _data/banking.yml [hunk 2]
```diff
@@ -234,6 +234,14 @@ websites:
     facebook: bbvacompass
     twitter: BBVACompass
 
+  - name: Bendigo Bank
+    url: https://www.bendigobank.com.au
+    img: bendigobank.png
+    facebook: BendigoBank
```
Comment (Carlgo11) [inline _data/banking.yml]: Whoops, seems to be an extra space here 😅 
```suggestion
    facebook: BendigoBank
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**